### ISSUE TITLE
Not working on Phonegap (added manually)

### ISSUE THREAD

**Issue by 1fancy** [tkyaji/cordova-plugin-crypt-file#87]
Hello,
When I've added the plugin manually to my app:
<plugin name="cordova-plugin-crypt-file" />

Nothing happend, on debug and release builds ?
How can I activate the plugin ?

Kind Regards.

**Comment by pratikchourdia** [comment]
Exactly, Added Plugin in config file but nothing happen. Phonegap is using version 1.2.0 But no effect on Apks.. Please give any way to work it.

Regards,

**Comment by mikael-onsilva** [comment]
Same here.

**Comment by lilmnm-kamikaze-** [comment]
if you are using phonegap build it will not work. This plugin uses hooks which phonegap build does not support.

**Comment by mikael-onsilva** [comment]
Yeah, I've figured that out after searching a little bit. Yet, I'm still not being able to work this out creating the project via CLI and building w Android Studio (I've got once, but now isn't working anymore). Any special configs to make it work?
